cli/internal/helm/charts/kms/values.yaml

```diff
@@ -0,0 +1,22 @@
+# Namespace to which KMS will be deployed.
+namespace: "kube-system"
+# Port on which the service will listen.
+kmsPort: 9000
+# Name of the ConfigMap that holds measurements and other info.
+joinConfigCMName: join-config
+# Path to which secrets/CMs are mounted.
+serviceBasePath: /var/config
```

Comment:
I thought it would be best if we controlled the mount path from the go code. The code that does the deployment sets this values based on the go constant. 
Removing the key from the values file but keeping it configurable seems counter-intuitive to me.